lib/tools/adb-commands.js

```diff
@@ -674,4 +674,37 @@ methods.getDeviceProductLocale = async function () {
   return await this.getDeviceProperty("ro.product.locale");
 };
 
+methods.getModel = async function () {
+  let stdout = await this.shell('getprop');
+  let model = new RegExp(/ro.product.model]:([^\r?\n]+)*/g).exec(stdout);
+  if (model && model.length >= 2) {
```

Comment:
Can you also add a comment about what the replace is doing? Otherwise one needs to examine it to know that you're removing square brackets.